exercises/practice/pov/.meta/example.ex

```diff
@@ -0,0 +1,133 @@
+defmodule Pov do
+  # Structs and types
+
+  @typedoc """
+  A tree, which is made of a node with several branches
+  """
+  @type tree :: {any, [tree]}
+
+  defmodule Zipper do
+    defstruct [:focus, genealogy: []]
+    @type t :: %Zipper{focus: Pov.tree(), genealogy: [Crumb.t()]}
```

Comment:
Dialyzer is complaining that it doesn't know the type `Crumb.t`. I think that can be fixed by first defining the Crumb module, and only then the Zipper module 🤔